src/Components/common/validateFormTestimonials.js

```diff
@@ -0,0 +1,18 @@
+export const validateFormTestimonials = (values) => {
+    let errors={}
+
+    if(!values.image) errors.image='You need to add a image';
+
+    if(!values.name) errors.name='You need to enter a name';
+    if(values.name.length<4) errors.name='The name must have more than 4 letters';
+    
+    if(!values.description) errors.description='You need to add a description';
+    
+    return errors
+
+}
+
+export const replaceCKeditor = (values)=>{
```

Comment:
this is not a validation function so it shoudl be on  ckEditor file. Also, im pretty sure that it do not replaces the ckEditor for something, it setDescriptionValue. And, because description its specific of the testimonial it should go on TestimonialForm file. If you can do it a little more generic, it should be on a ckEditor file.

Something more generic could be something like
```
export const setCKEditorText = (values, field) => {
    values[field] = values[field].replace(/'<p>'|['</p>']/gi,'');
    return values;
}
```